openwisp-monitoring/files/monitoring.agent

```diff
@@ -118,6 +118,7 @@ handle_sigusr1() {
 }
 
 send_data() {
+	success_count=0
```

Comment:
If we start from zero, the pause will not be executed after 10 requests, but 11, right? So If we want the implementation to follow the spec and if we want the comment below to be technically correct, I think we should start from 1 here.

Reply:
MB you're right. Will update it!